scripts/DMF/dmfsnmp/eaton-pdu-marlin-mib.dmf

```diff
@@ -1,41 +1,10 @@
 <?xml version="1.0" ?>
 <nut version="1.0.0" xmlns="http://www.networkupstools.org/dmf/snmp/snmp-ups">
-	<lookup name="marlin_threshold_voltage_alarms_info">
```

Comment:
Strange how the DMF's contents got shuffled :\ Can this be somehow build-locale dependent?

Reply:
can't say. Another mystery of life ;)